src/new-client/src/App/Trades/TradesGrid/TradesGrid.tsx

```diff
@@ -40,20 +41,21 @@ const highlightBackgroundColor = css`
 `
 
 const TableBodyRow = styled.tr<{ pending?: boolean; highlight?: boolean }>`
-  &:nth-child(even) {
-    background-color: ${({ theme }) => theme.core.darkBackground};
-  }
-  &:hover {
-    background-color: ${({ theme }) => theme.core.alternateBackground};
-  }
   height: 2rem;
   ${({ highlight }) => highlight && highlightBackgroundColor}
 `
 
-const TableBodyCell = styled.td<{ numeric?: boolean; rejected?: boolean }>`
+const TableBodyCell = styled.td<{
+  numeric?: boolean
+  rejected?: boolean
+  relativeWidth: number
+}>`
   text-align: ${({ numeric }) => (numeric ? "right" : "left")};
   padding-right: ${({ numeric }) => (numeric ? "1.6rem;" : "0.1rem;")};
   position: relative;
+  width: ${({ relativeWidth }) => `${relativeWidth}vw`};
+  min-width: ${({ relativeWidth }) => `${relativeWidth * 11.5}px`};
```

Comment:
Where does 11.5 come from?

Reply:
**Not necesary anymore**, fixed. 

The value originally was 12.5 instead of 11.5, and I took that from the proportional value I had in the colConfig.js, there we had the width values in pixels so I converted the proportion of each value in vw (which maybe wasn't the best approach since the vw will take the entire window and not the parent div), for example, if all columns occupied 1000px and the fist one had a witdth of 100px, taking a 10% of the total, the vw relation would be of 10vw and so on, then we would do the convertion again from vw to px in the case of min-width **luckly this is not necesary anymore**. Also, it went from 11.5 to 12.5 by adjusting the values "By eye" (i.e by trial and error). 

As for the width, I kept this value since it's the only way I found that would allow me to keep similar widths between the header of the table and the cells within the rows.